install.sh

```diff
@@ -7,7 +7,13 @@ conda install -c anaconda pip
 # Install requirements
 pip install -r requirements.txt
 
+# Install pytorch to support GPU on mac m1
+# Expect the M1-GPU support to be included in the next stable release. For the time being, it only is found in the Nightly release.
+conda install pytorch -c pytorch-nightly
```

Comment:
We can't have a nightly version as it will change and potentially break codebase in a few months/years time. There need to be clear versioning as  this repository is not meant to be continuously maintained.

Reply:
also why is it necessary to install it through conda?